File: Target.App/ClientApp/src/app/model/lancamento.ts
export enum Status {
  Valido = 0,
  Cancelado = 1,
}

export class Lancamento {
  id: string = '';
  descricao: string = '';
  valor: number = 0;
  data: Date = new Date();
  status: Status = Status.Valido;
  avulso: boolean = false;
}

export class LancamentoList {
  lancamentos: Array<Lancamento> = [];
  total: number = 0;
}

File: Target.App/Model/Lancamento.cs
namespace Target.App.Model
{
    public class Lancamento
    {
        public Guid Id { get; set; }
        public string Descricao { get; set; }
        public DateTime Data { get; set; }
        public double Valor { get; set; }
        public Status Status { get; set; }
        public bool Avulso { get; set; }
    }
}
